--- makepkg	2018-08-28 21:55:13.766161019 +0800
+++ makepkg.after	2018-08-28 21:59:54.526188987 +0800
@@ -180,8 +180,9 @@
 }
 
 enter_fakeroot() {
+	export PATH=$HOME/bin:$PATH
 	msg "$(gettext "Entering %s environment...")" "fakeroot"
-	fakeroot -- $0 -F "${ARGLIST[@]}" || exit $?
+	fakeroot $0 -F "${ARGLIST[@]}" || exit $?
 }
 
 # Automatically update pkgver variable if a pkgver() function is provided
@@ -1413,12 +1414,12 @@
 	if (( EUID == 0 )); then
 		error "$(gettext "Running %s as root is not allowed as it can cause permanent,\n\
 catastrophic damage to your system.")" "makepkg"
-		exit $E_ROOT
+		#exit $E_ROOT
 	fi
 else
 	if [[ -z $FAKEROOTKEY ]]; then
 		error "$(gettext "Do not use the %s option. This option is only for internal use by %s.")" "'-F'" "makepkg"
-		exit $E_INVALID_OPTION
+		#exit $E_INVALID_OPTION
 	fi
 fi

